lib/create-app-definition-api.ts

```diff
@@ -177,19 +177,14 @@ export default function createAppDefinitionApi(makeRequest: MakeRequest) {
      * .catch(console.error)
      * ```
      */
-    getInstallationsForOrg({
-      organizationId,
-      appDefinitionId,
-    }: {
-      organizationId: string
-      appDefinitionId: string
-    }) {
+    getInstallationsForOrg() {
+      const raw = this.toPlainObject() as AppDefinitionProps
       return makeRequest({
         entityType: 'AppDefinition',
         action: 'getInstallationsForOrg',
         params: {
-          appDefinitionId,
-          organizationId,
+          appDefinitionId: raw.sys.id,
```

Comment:
out of curiosity, where is the appDefinitionId coming from when using in conjunction with the app-sdk? 
Does the adapter set it automatically?